src/renderer/services/wallets.js

```diff
@@ -194,6 +578,234 @@ export default {
     });
   },
 
+  getClient() {
+    // note use `bwsurl` all lowercase;
+    const bwc = new BWC({
+      baseUrl: 'https://bws.bitpay.com/bws/api',
+      verbose: false,
+      timeout: 100000,
+      transports: ['polling'],
+    });
+
+    return bwc;
+  },
+
+  getKey() {
+    return BWC.Key;
+  },
+
+  generateBTCETHWallet(walletName) {
+    return new Promise(async (resolve, reject) => {
+      const opts = {
+        name: walletName,
+        m: 1,
+        n: 1,
+        myName: 'me',
+        bwsurl: 'https://bws.bitpay.com/bws/api',
+        singleAddress: false,
+        coin: 'btc',
+      };
+      const Key = this.getKey();
+
+      const lang = {
+        name: 'English',
+        isoCode: 'en',
+      };
+
+      // creating key
+
+      const key = Key.create({
+        lang // eslint-disable-line comma-dangle
+      });
+
+      const btcTestnet = await this.createTestnetBTCWallet(key, opts);
+      const btcMainnet = await this.createMainnetBTCWallet(key, opts);
+
+      if (btcTestnet.err || btcMainnet.err) {
+        reject({
+          err: true,
+        });
+      } else {
+        resolve({
+          key,
+          btcTestnetWalletClient: btcTestnet.walletClient,
+          btcMainnetWalletClient: btcMainnet.walletClient,
+        });
+      }
+    });
+  },
+
+  generateBTCwalletKey() {
+    const Key = this.getKey();
+
+    const lang = {
+      name: 'English',
+      isoCode: 'en',
+    };
+
+    // creating key
+
+    const key = Key.create({
+      lang // eslint-disable-line comma-dangle
```

Comment:
nit: just put the comma instead of disabling the lint rule

Reply:
Done added